src/renderer/pages/Basket/model/basket-page-model.ts

```diff
@@ -1,35 +1,108 @@
-import { combine, createEvent, createStore, restore, sample } from 'effector';
+import { combine, createEffect, createEvent, createStore, sample, scopeBind } from 'effector';
 
 import { walletModel } from '@entities/wallet';
 import { basketModel } from '@entities/basket';
-import { includes } from '@shared/lib/utils';
-import { ID } from '@shared/core';
-import { getTransactionTitle } from '@entities/transaction';
+import { BasketTransaction, ID, TransactionType } from '@shared/core';
+import { TransferTypes, XcmTypes } from '@entities/transaction';
+import {
+  transferValidateModel,
+  addProxyValidateModel,
+  addPureProxiedValidateModel,
+  removeProxyValidateModel,
+  removePureProxiedValidateModel,
+  bondNominateValidateModel,
+  payeeValidateModel,
+  nominateValidateModel,
+  bondExtraValidateModel,
+  restakeValidateModel,
+  unstakeValidateModel,
+  withdrawValidateModel,
+} from '@features/operations/OperationsValidation';
+import { networkModel } from '@entities/network';
 
-const queryChanged = createEvent<string>();
 const txSelected = createEvent<ID>();
 const allSelected = createEvent();
 
 const $selectedTxs = createStore<ID[]>([]);
-const $query = restore(queryChanged, '');
+const $invalidTxs = createStore<Set<ID>>(new Set());
+
+const validateFx = createEffect((transactions: BasketTransaction[]) => {
+  const validateTransferBound = scopeBind(transferValidateModel.events.validationStarted, { safe: true });
+  const addProxyValidateBound = scopeBind(addProxyValidateModel.events.validationStarted, { safe: true });
+  const addPureProxiedValidateBound = scopeBind(addPureProxiedValidateModel.events.validationStarted, { safe: true });
+  const removeProxyValidateBound = scopeBind(removeProxyValidateModel.events.validationStarted, { safe: true });
+  const removePureProxiedValidateBound = scopeBind(removePureProxiedValidateModel.events.validationStarted, {
+    safe: true,
+  });
+  const bondNominateValidateBound = scopeBind(bondNominateValidateModel.events.validationStarted, { safe: true });
+  const nominateValidateBound = scopeBind(nominateValidateModel.events.validationStarted, { safe: true });
+  const bondExtraValidateBound = scopeBind(bondExtraValidateModel.events.validationStarted, { safe: true });
+  const payeeValidateBound = scopeBind(payeeValidateModel.events.validationStarted, { safe: true });
+  const restakeValidateBound = scopeBind(restakeValidateModel.events.validationStarted, { safe: true });
+  const unstakeValidateBound = scopeBind(unstakeValidateModel.events.validationStarted, { safe: true });
+  const withdrawValidateBound = scopeBind(withdrawValidateModel.events.validationStarted, { safe: true });
+
+  for (const tx of transactions) {
+    if (TransferTypes.includes(tx.coreTx.type) || XcmTypes.includes(tx.coreTx.type)) {
+      validateTransferBound({ id: tx.id, transaction: tx.coreTx });
+    }
+
+    if (tx.coreTx.type === TransactionType.ADD_PROXY) {
```

Comment:
To avoid checking all conditions even when the first one has passed, we can use object:
```javascript
const TransactionValidators = {
  [TransactionType.ADD_PROXY]: addProxyValidateBound,
```
`TransferTypes.includes(tx.coreTx.type) || XcmTypes.includes(tx.coreTx.type)` is a special one, can leave it as is